README-SI.md

```diff
@@ -181,4 +178,4 @@ GitHub හි දායක වීමට සුදුසු issues වෙත ස
 
 ## බලපත්‍රය
 
-<a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://licensebuttons.net/l/by-sa/4.0/88x31.png" /></a><br />මෙම ව්‍යාපෘතිය <a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a> යටතේ බලපත්‍රගත කර ඇත.

```

Comment:
This can be kept and added back in.

Reply:
https://github.com/freeCodeCamp/how-to-contribute-to-open-source/commit/79ca1ff547f29c521f8558fd89c6ec48f6ba069f